app/assets/stylesheets/base/variables.scss

```diff
@@ -111,5 +111,8 @@ $wilkinson-primary: #008ec1;
 $wilkinson-secondary: #209cc9;
 $wilkinson-tertiary: #007ba7;
 
+$law-primary: #002f7c;
+$law-secondary: #01235b;
+
```

Comment:
According to Wai, `primary` is the main color like for the ribbon, `secondary` is the hover color, and `tertiary` is the active color.